ols-2.md

```diff
@@ -32,25 +32,26 @@ photos:
 {% assign all-speakers = '' %}
 {% assign all-hosts = '' %}
 {% for w in schedule.weeks %}
-    {% for c in w[1].calls %}
-        {% if c.type == 'Cohort' %}
-            {% for r in c.resources %}
-                {% if r.type == 'slides' and r.speaker %}
-                    {% capture all-speakers %}{{ all-speakers}}, {{ r.speaker }}{% endcapture %}
-                {% endif %}
-            {% endfor %}
-        {% endif %}
-        {% if c.hosts %}
-            {% for h in c.hosts %}
-                {% capture all-hosts %}{{ all-hosts}}, {{ h }}{% endcapture %}
-            {% endfor %}
-        {% endif %}
-    {% endfor %}
+{% for c in w[1].calls %}
+{% if c.type == 'Cohort' %}
+{% for r in c.resources %}
+{% if r.type == 'slides' and r.speaker %}
+{% capture all-speakers %}{{ all-speakers}}, {{ r.speaker }}{% endcapture %}
+{% endif %}
+{% endfor %}
+{% endif %}
+{% if c.hosts %}
+{% for h in c.hosts %}
+{% capture all-hosts %}{{ all-hosts}}, {{ h }}{% endcapture %}
+{% endfor %}
+{% endif %}
+{% endfor %}
 {% endfor %}
 {% assign p-speakers = all-speakers | remove_first: ', ' | split: ", " | uniq | sort %}
 {% assign p-hosts = all-hosts | remove_first: ', ' | split: ", " | uniq | sort %}
 
 # The OLS-2 program
+
```

Comment:
This empty line should stay so the `{:.no_toc}` is applied to it

Reply:
okay